packages/backend/src/apps/todoist/triggers/get-tasks/get-tasks.ts

```diff
@@ -0,0 +1,30 @@
+import { IGlobalVariable } from '@automatisch/types';
+
+const getActiveTasks = async ($: IGlobalVariable) => {
+
+  let requestPath = `/tasks`;
+
+  requestPath += '?project_id=' + ($.step.parameters.projectId as string).trim();
+  requestPath += '&section_id=' + ($.step.parameters.sectionId as string).trim();
+  requestPath += '&label=' + ($.step.parameters.label as string).trim();
```

Comment:
@irlevesque, I could not manage to get tasks by "label". I provided `label name` as a value (I even tried it surrounded with quotes) and tested it. However, it doesn't give an active task with the label even though I see such tasks in the API response when I do not try to filter it by `label`.

I wonder if you could help me with testing this.

Reply:
I see that I was using something called "shared label" and it wasn't working. Now, I tried it with a personal label and it filters by `label`! I am fine to resolve this conversation upon this.